src/styles/index.scss

```diff
@@ -1,3 +1,67 @@
+@import './vars';
+@import './mixins';
+@import './keyframes';
+
 body {
   margin: 0;
+  height: 100vh;
+
+  @include stopwatch-flex-property;
+}
+
+.stopwatch {
+  width: $stopwatch-size;
+  height: $stopwatch-size;
+  color: #000;
+
+  @include stopwatch-flex-property;
+  @include stopwatch-border-style(1vmin, dotted, 50%);
+
+  &__hand {
+    bottom: 50%;
+    transform-origin: bottom;
+    position: absolute;
+
+    @include stopwatch-flex-property;
```

Comment:
It's a good practice to avoid using absolute positioning unless it's necessary. Absolute positioning takes the element out of the normal document flow which can lead to overlapping of elements and other issues. Try to use other CSS properties like flexbox or grid to position your elements.